### ISSUE TITLE
bug: (rln-relay) chat2 displays zero'd keys

### ISSUE THREAD

**Issue by fryorcraken** [status-im/nwaku#1221]
### Problem

When starting `chat2` with existing RLN credentials, it shows the following message:

```
 setting up waku-rln-relay in on-chain mode... 
your membership index is: 0
your rln identity key is: 0000000000000000000000000000000000000000000000000000000000000000
your rln identity commitment key is: 0000000000000000000000000000000000000000000000000000000000000000
```

Instead of the passed RLN Credentials

### Impact

Confusion for the user as message sent seem to have a proof attached:

```
>> hello there
witness generation took: 24.97ms
proof generation took: 99.30ms
--rln epoch: 166486172
<Oct 04, 16:35> nwaku-fr: hello there
```

### To reproduce

```
./build/chat2 --fleet=none \
    --content-topic=/toy-chat/2/luzhou/proto \
    --ports-shift=1 \
    --rln-relay=true \
    --rln-relay-dynamic=true \
    --rln-relay-eth-contract-address=0x367F3e869cF2E1DD28644E308b42652cb6fcDA72 \
    --staticnode="/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAkyQAiEgZgX899dHFgjdooCUXN2zsQPHcy9D6LbX3SCEXA" \
    --rln-relay-eth-client-address=wss://goerli.infura.io/ws/v3/<use your own token please> \
    --rln-relay-membership-index=53 \
    --rln-relay-id-key=<use your own creds please> \
    --rln-relay-id-commitment-key=40ece25c725b15423b1d1acc299753fe196cdd593434c8025876dcf6de84c62a \
    --rln-relay-eth-account-address:0xC4E45C1423Ebf3Ab780a2DACf687c5Dfb2cC3594
```

### nwaku version/commit hash
787464487ba9331f8b468cf666b47be26ceeb181

Built with `RLNZEROKIT=true`

**Comment by staheri14** [comment]
> --rln-relay-membership-index=53 \
    --rln-relay-id-key=<use your own creds please> \
    --rln-relay-id-commitment-key=40ece25c725b15423b1d1acc299753fe196cdd593434c8025876dcf6de84c62a 


These set of parameters are deprecated, the rln credential is supposed to be re-used using the credential file path `--rln-relay-cred-path`.

**Comment by staheri14** [comment]
Can you please check it with the credential path and let me know if there is still any issue? We shall remove those config options to avoid confusion.

**Comment by staheri14** [comment]
The fix would be to remove those config options i.e., 
``` --rln-relay-membership-index 
    --rln-relay-id-key 
    --rln-relay-id-commitment-key 
```

**Comment by staheri14** [comment]
No need to generate, you only need to pass a path to the file, the code will generate it for you. e.g., `--rln-relay-cred-path=.` which is the root directory.

**Comment by staheri14** [comment]
> Shall it be removed then?

Yes, for sure.

**Comment by fryorcraken** [comment]
> No need to generate, you only need to pass a path to the file, the code will generate it for you. e.g., `--rln-relay-cred-path=.` which is the root directory. Please also see this [`master`/docs/tutorial/onchain-rln-relay-chat2.md#how-to-persist-and-reuse-rln-credential](https://github.com/status-im/nwaku/blob/master/docs/tutorial/onchain-rln-relay-chat2.md?rgh-link-date=2022-10-05T17%3A49%3A09Z#how-to-persist-and-reuse-rln-credential)

Is there an RFC for this format (see https://github.com/vacp2p/rfc/issues/543)?

**Comment by rymnc** [comment]
Fixed in #1258
